src/popup/components/TransferReview.vue

```diff
@@ -175,12 +174,12 @@ export default defineComponent({
             amount,
             recipient,
             hash: actionResult.hash,
-            type: 'spendToken',
+            type: 'SpendTx',
             pendingTokenTx: true,
             tx: {
               callerId: account.value.address,
               contractId: selectedAsset.contractId,
-              type: SCHEMA.TX_TYPE.contractCall,
+              type: 'ContractCallTx',
```

Comment:
there is a way to convert TS Enum to string
```suggestion
              type: Tag[Tag.ContractCallTx],
```

Reply:
Done. Resolved